test_scripts/Defects/5_0/2441_SDL_sending_redundant_UnregisterAppInterface_notification.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2441
+--
+-- Description:
+-- SDL sending redundant UnregisterAppInterface notification to mobile.
+-- Steps to reproduce:
+-- 1) Register and activate App
+-- 2) Unregister App
+-- Expected:
+-- 1) SDL must send response to mobile applications request and send notification to HMI. 
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local variables ]]
+
+-- [[ Local function ]]
+local function unregisterApp()
+    local cid = common.getMobileSession():SendRPC("UnregisterAppInterface", {})
+    common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered")
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+    common.getMobileSession():ExpectNotification("UnregisterAppInterface")
```

Comment:
@TMyza Please check right notification instead wrong one here (`UnregisterAppInterface` is not a notification, it is request/response)
Current check is incorrect. Please fix it according Mobile_API.